src/database/models/eventsModel.js

```diff
@@ -0,0 +1,35 @@
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+//Event model consisting of the event title, start and end time, and checkIn code
+var eventSchema = new Schema(
```

Comment:
@Alacau same here

Reply:
event title, event date, event location, start time, end time, check-in code, background image

im not sure if we need an event description @leestanley